packages/esm-patient-medications-app/src/components/medications-details-table.component.tsx

```diff
@@ -128,26 +130,31 @@ const MedicationsDetailsTable: React.FC<ActiveMedicationsProps> = ({
             </p>
           </div>
           <p className={styles.bodyLong01}>
-            {medication.orderReasonNonCoded ? (
+            {medication.orderReasonNonCoded && (
               <span>
                 <span className={styles.label01}>{t('indication', 'Indication').toUpperCase()}</span>{' '}
                 {medication.orderReasonNonCoded}
               </span>
-            ) : null}
-            {medication.quantity ? (
+            )}
+            {medication.quantity && (
               <span>
                 <span className={styles.label01}> &mdash; {t('quantity', 'Quantity').toUpperCase()}</span>{' '}
                 {medication.quantity} {medication?.quantityUnits?.display}
               </span>
-            ) : null}
-            {medication.dateStopped ? (
+            )}
+            {(medication.dateStopped || medication.autoStopDate) && (
```

Comment:
Why the inclusion of `medication.autoStopDate` here?

Reply:
@denniskigen It was one of the comments left by Ian on this PR https://github.com/openmrs/openmrs-esm-patient-chart/pull/1691 and I Quote  `@[ibacher](https://github.com/ibacher) ibacher [on Jul 5](https://github.com/openmrs/openmrs-esm-patient-chart/pull/1691#discussion_r1667011209)
I don't know that it's guaranteed that dateStopped will always be set. That is, it will be set for medications that have been discontinued, but maybe not for those with a autoStopDate (sp. of property).`

Reply:
However, if we're adding it to the condition, it should also be added to the tooltip.